lib/ui/components/scenario-overview.js

```diff
@@ -47,6 +54,33 @@ export class ScenarioOverview extends Component {
     );
   }
 
+  onToggleRequest(requestId) {
+    this.props.toggleMock(this.props.id, requestId);
+  }
+
+  _renderMasterToggle() {
+    const toggleAllRequests = () => {
```

Comment:
this should be 2 separate functions, renderMasterToggle and toggleAllRequests, there's no reason for them to live in the same function